visualization/app/codeCharta/state/store/fileSettings/blacklist/blacklist.service.ts

```diff
@@ -21,6 +25,17 @@ export class BlacklistService implements StoreSubscriber {
 		}
 	}
 
+	public onFileStatesChanged(fileStates: FileState[]) {
+		this.reset(fileStates)
+	}
+
+	public reset(fileStates: FileState[]) {
+		const visibleFiles = FileStateHelper.getVisibleFileStates(fileStates).map(x => x.file)
+		const withUpdatedPath = !!FileStateHelper.isPartialState(fileStates)
```

Comment:
Doesn't `isPartialState´ return a boolean? Why would you negate twice here?

Reply:
Yea that actually doesn't make sense :D